client/app/bundles/course/forum/handles.ts

```diff
@@ -15,10 +24,25 @@ const getTopicTitle = async (
 };
 
 export const forumHandle: DataHandle = (match) => {
+  const courseId = match.params.courseId;
+
+  return {
+    getData: async (): Promise<CrumbPath> => {
+      const { data } = await CourseAPI.forum.forums.index();
+
+      return {
+        activePath: `/courses/${courseId}/forums`,
+        content: { title: data.forumTitle ?? translations.header },
```

Comment:
Wrong operator `??`. Should use `||`. Otherwise it doesn't handle empty string.  
Go double-check the rest.

Reply:
should be done now

Reply:
I just tried again, how come the top-level Forum breadcrumb is not clickable when I am in a nested forum/topic?

Reply:
fixed.. after learning again how Breadcrumbs is being implemented, apparently I need to provide the `url` field towards the content as well